style.css

```diff
@@ -0,0 +1,11 @@
+@media only screen and (max-width: 800px) {
+    body {
+      background-color: lightpink;
+    }
+  }
+  
+  #main-doc{
+    position:reltive;
```

Comment:
Nice job pointing this out, as these errors will cause trouble later on when debugging :+1: However, It would be better if you could make suggestions instead of giving an order to sound nicer.